torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1404,6 +1404,104 @@ def safe_expand(r):
     else:
         return r
 
+
+@lru_cache(None)
+def _maybe_evaluate_static_worker(
+    expr: sympy.Expr,
+    symbol_info: Tuple[Tuple[sympy.Symbol, ValueRanges, sympy.Integer, bool], ...],
```

Comment:
Maybe use a named tuple (or at least add a comment) to indicate what the individual fields represent?